zerver/openapi/zulip.yaml

```diff
@@ -9254,6 +9254,14 @@ paths:
                   - $ref: "#/components/schemas/InvalidApiKeyError"
                   - $ref: "#/components/schemas/MissingArgumentError"
                   - $ref: "#/components/schemas/UserNotAuthorizedError"
+        "403":
+          description: |
+            Forbidden
```

Comment:
This should comment on the user account being deactivated.

Reply:
Oh, hadn't realized the level this was at -- I think we just was `Forbidden.` to match the "Bad request" styling above? 

(I don't think these end up getting displayed).

Reply:
> (I don't think these end up getting displayed).

Yeah for responses their description is not displayed only example is displayed.